bokeh/models/annotations.py

```diff
@@ -530,6 +530,52 @@ class BoxAnnotation(Annotation):
 
     """)
 
+class FilledArea(Annotation):
+    ''' Render a filled area band along a dimension.
+
+    '''
+    lower = NumberSpec(help="""
+    The lower coordinates to locate the text anchors.
+    """)
+
+    upper = NumberSpec(help="""
+    The upper coordinates to locate the text anchors.
+    """)
+
+    coords = NumberSpec(help="""
```

Comment:
Don't have to worry about not but worth considering the case where upper and lower are different lengths... I think that implies multiple sources but may theres just a new different annotation for that.